app/controllers/applicants_controller.rb

```diff
@@ -13,6 +13,13 @@ def index
     end
   end
 
+  def first_timers
```

Comment:
Feel like this should be on the index action as a custom filter, but I remember some of my changes enabling nested resources earlier added complexity in that action, so it's probably getting to be time to revisit that issue.